### DISCUSSION TITLE
Not able to use `bg-neutral-950` after adding daisy

### DISCUSSION THREAD

**Discussion by arjoonn-s** [saadeghi/daisyui#2419] [Q&A] [answered]
I ran into a strange problem today:

- I'm using jinja templates to generate my HTML and the string `bg-neutral-950` appears in my source file as it is.
- All other classes for that file are being generated correctly.
- When I add daisy UI to my plugins list, I'm no longer able to use this tailwind colors class. It gets removed from the final output.

```
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "../api/src/templates/**/*.{html,js}",
    "../site/src/**/*.{html,js}",
  ],
  theme: {
    extend: {},
  },
  plugins: [
    require("daisyui"),
    require('@tailwindcss/typography'),
    require('@tailwindcss/forms')({
      strategy: "class",
    }),
    //require('@tailwindcss/aspect-ratio'),
    //require('@tailwindcss/container-queries'),
  ]
}
```

Removing daisy from the plugin list makes the color work without issues.
What am I doing wrong? This last section in daisy docs seems to suggest that I should be able to use original tailwind color names?

https://daisyui.com/docs/colors/#-3

**Comment by saadeghi** [comment, marked as answer]
It was a bug. Thanks for reporting.  
It's fixed now in daisyUI `3.9.3`

**Reply by arjoonn-s** [reply]
Oh this was such a good fix :heart: 

Thank you so much to whoever fixed that bug :tada:
